### ISSUE TITLE
Only IPs destination displayed ?

### ISSUE THREAD

**Issue by SkYNewZ** [objective-see/LuLu#375]
On each received alerts, I can only see IPV4/IPV6 destination. I guess it's not exclusively from LuLu but I'd like to know if anyone knows how to display URLs instead of IPs.

<img width="1287" alt="image" src="https://user-images.githubusercontent.com/26750012/122741866-4e387580-d27d-11eb-9b11-0c437f03c0ee.png">

**Comment by artkiko** [comment]
I uninstalled and installed 1.2.3 and I got the urls back.

**Comment by artkiko** [comment]
Correct. 2.4.0 and 2.4.1 do not display urls for me. 1.2.3 does. Haven't tried the intermediate versions but that's the only workaround I have for now. If you figure this out with a current version, please share.
